backend/src/main/java/com/huellapositiva/infrastructure/orm/entities/JpaVolunteerProposal.java

```diff
@@ -0,0 +1,31 @@
+package com.huellapositiva.infrastructure.orm.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@IdClass(VolunteersProposals.class)
+@Table(name = "volunteers_proposals")
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class JpaVolunteerProposal implements Serializable {
+
+    @Id
+    @Column(name = "proposal_id")
+    private String proposal;
+
+    @Id
+    @JoinColumn(name = "volunteer_id", referencedColumnName = "id")
+    @OneToOne(optional = false, fetch = FetchType.LAZY)
+    private JpaVolunteer volunteer;
+
+    @Column(name = "confirmed")
+    private boolean confirmed;
```

Comment:
I am not sure that a boolean is enough. I can see we have three states:
Volunteer joins proposal -> status - pending
Volunteer is rejected -> status rejected
Volunteer is confirmed -> status confirmed

Reply:
I was talking to Juan. Juan wants there to only be two statuses: confirmed (by default in true) and rejected (false).

Reply:
Ok, we can leave it that way.
Just remember that the product team gives us functional requirements, not implementation details. We are always free to decide the most convenient implementation to fit the purposes we are given.
Functional state "Confirmed" could map to internal CONFIRMED
Functional state "Rejected" could map to internal REJECTED or PENDING